Compliance to awesome repository requirements (#38)
docker/awesome-compose: CONTRIBUTING.md

# Contributing

Contributions should be made via pull requests. Pull requests will be reviewed
by one or more maintainers and merged when acceptable.

The goal of the Awesome Compose is to provide a curated list of application 
samples that can be easily deployed with [Docker Compose](https://github.com/docker/compose).

## Missing an example? 

You can request a new example of an application by submitting an issue to our GitHub repository. 

Before submitting a new application, check if there isn't already application sample matching your need.

> If there is one, consider to update it instead of create a new one.

Reviewer: nit:
```suggestion
If there is one, consider updating it instead of creating a new one.
```

 
If you would like to submit a new application example, please start by submitting a proposal as an issue.
The maintainers will then use this issue to discuss what the the most valuable example for the application,
 technology, language, or framework would be.

> After the choice has been made, you can submit your a pull request not forgetting to:

Reviewer: ```suggestion
After the choice has been made, you can submit a pull request with the example remembering to:
```

# Contributing

Contributions should be made via pull requests. Pull requests will be reviewed
by one or more maintainers and merged when acceptable.

The goal of the Awesome Compose is to provide a curated list of application 
samples that can be easily deployed with [Docker Compose](https://github.com/docker/compose).

> ## Missing a kind of application? 

Reviewer: ```suggestion
## Missing an example?
```

> You can request a new example of application by submitting an issue to our GitHub Repository.  

Reviewer: nit: missing `\n` after heading

Reviewer: ```suggestion

You can request a new example of an application by submitting an issue to our GitHub repository.  
```

> If you would like to implement a new application sample, please submit an issue with a proposal first,

Reviewer: ```suggestion
If you would like to submit a new application example, please start by submitting a proposal as an issue.
```

to be sure that we can use it.   

> Then we'll consult the community to know what can be the most valuable application with this technology/language/framework...

Reviewer: ```suggestion
The maintainers will then use this issue to discuss what the the most valuable example for the application, technology, language, or framework would be.
```

> After the choice made, you can submit your pull request following the choice of the community without forgetting to add :

Reviewer: ```suggestion
After the choice has been made, you can submit your a pull request not forgetting to:
```

> - a local README.md to describe the application and explain how to run/use the sample.

Reviewer: ```suggestion
- include an example README.md to describe the application and explain how to run/use the sample
```

- edit the global README.md to add your sample in the repository main list.

> ## Update a existing sample

Reviewer: We can actually remove this

> Before submitting a new application, check if there isn't already application sample matching your need.

Reviewer: We should move this (and the next line) up to the other section

If there is one, consider to update it instead of create a new one.

To help maintainers understand what user or developer problem needs to be
solved, the first step to a contribution is usually submitting an issue. A well
written issue is one that clearly outlines the developer or user problem that
needs to be solved along with a list of requirements for resolution of the
problem. If there are multiple possible solutions to the problem, these can be
outlined in the issue. Once consensus is reached on how to resolve the issue, a
pull request can be created.

Pull requests that propose minor changes or improvements may be submitted
without an associated issue or discussion.

For large or high impact changes, contributors can reach out to maintainers
before starting work. This will ensure that contributors and maintainers are

> aligned and increase the chance that the change is accepted.

Reviewer: I think you can remove this as it's more targeted at code/specs
